swc/repositories.bzl

```diff
@@ -98,7 +98,7 @@ def _determine_version(rctx):
 
 def _swc_repo_impl(repository_ctx):
     version = _determine_version(repository_ctx)
-    if not versions.is_at_least("1.3.25", version.lstrip("v")):
+    if not versions.is_at_least("1.3.41", version.lstrip("v")):
```

Comment:
we could make the minimum version conditional on whether you're on windows.
Or just leave it alone, the tiny number of windows users can discover the need to upgrade by reading the release notes.